src/Lib/Managers/NobManager.vala

```diff
@@ -509,6 +549,7 @@ public class Akira.Lib.Managers.NobManager : Object {
         for (int i = 0; i < 9; i++) {
             var nob = new Selection.Nob (root, (Managers.NobManager.Nob) i);
             nob.set ("visibility", Goo.CanvasItemVisibility.HIDDEN);
+            nob.pointer_events = Goo.CanvasPointerEvents.NONE;
```

Comment:
We should add a comment here to explain why we're preventing pointer events on the nobs and the way we're handling nobs selection.

Reply:
Added a comment